packages/ds-healthcare-gov/src/styles/components/_Accordion.scss

```diff
@@ -1,10 +1,8 @@
 @use '../layout' as *;
 
-:root {
+.ds-c-accordion {
   --accordion-icon-color: #0071bb;
```

Comment:
browsers didn't recognize this custom prop as having a value. moving it within the accordion class fixed that.

Reply:
this has been broken for some time. i don't know if this is the correct fill color hgov wants, but it seems unusual to me that they have a one-off hex for something like this (not seeing this in [available system colors](https://design.cms.gov/foundation/system-color-tokens/?theme=healthcare)).

Reply:
Weird, yeah, given that this has been broken for a while, I don't see any harm in setting it to `--accordion__background-color` instead when we fix it.

Reply:
Selected component interactions and links I believe use `--color-primary` the accordion should follow the same convention.  Nice find @zarahzachz 👍🏼